int16 hsv_lh
int16 hsv_ls
int16 hsv_lv
int16 hsv_hh
int16 hsv_hs
int16 hsv_hv
int16 clear_es
int16 clear_et
int16 clear_ds
int16 clear_dt
int16 clear_th
int16 roi_x1
int16 roi_y1
int16 roi_w1
int16 roi_h1
int16 roi_x2
int16 roi_y2
int16 roi_w2
int16 roi_h2
string id
